kuma/wiki/tasks.py

```diff
@@ -298,6 +298,17 @@ def build_locale_sitemap(locale):
         os.makedirs(directory)
 
     queryset = Document.objects.filter_for_list(locale=locale)
+    # Exclude any document whose slug starts with one of these prefixes:
+    #     - "Experiment:":
+    #         these documents are disallowed by robots.txt
+    #     - "Mozilla/QA/Marionette/WebDriver/status":
+    #         this document is a hard-wired redirect to an external domain, so
+    #         it can't be excluded as the content-based redirects are by the
+    #         "filter_for_list" method above
+    for slug_prefix in ('Experiment:',
```

Comment:
I think ``Experiment:`` should be moved into the default filters in ``filter_for_list``. In general, these URLs are implementation details for content swapping and shouldn't show up in list views like [document errors](https://developer.mozilla.org/en-US/docs/with-errors).

Reply:
I had wondered about this myself, so I like it much better, thanks @jwhitlock!